website/docs/latest/algorithms/regression/ridge.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: doc-page
+title: Ridge Regression
+---
+
+### About
+
+The `Ridge` regressor in Mahout implements a _closed-form_ solution to Ridge Regression (https://en.wikipedia.org/wiki/Tikhonov_regularization).
+
+
```

Comment:
If you have some time I might explain a little more about what Ridge regression is / when its appropriate to use, etc. This is good, and you have a wiki link which is also nice, I'm just picking nits.